src/Metric.php

```diff
@@ -92,6 +101,38 @@ public function scan($uri, \DOMXPath $xpath, $depth, Page $page, Metrics $contex
     }
 
     /**
+     * Get the machine name for a given error message
+     * 
+     * @param $error_message
+     * @return string
+     */
+    public function getMarkNameFromMessage($error_message)
+    {
+        switch ($error_message) {
```

Comment:
This is very hard to follow. It doesn't look like you are switching based on the value of `$error_message`, but rather the result of an expression `switch (true)` ?

Also the ternary `? true : false` is unecessary.